lib/mpl_toolkits/tests/test_axisartist_axis_artist.py

```diff
@@ -97,3 +99,7 @@ def test_axis_artist():
     axisline.label.set_pad(5)
 
     ax.set_ylabel("Test")
+
+    tightbbox = axisline.get_tightbbox()
+    assert math.isclose(tightbbox.height, 15.880208333333329)
+    assert math.isclose(tightbbox.width, 518.0625)
```

Comment:
Can these values be understood or do you just pin the current values?

Reply:
The latter. Although I guess one can probably figure them out.

Looking at similar tests there is the whole range from pinned values to computed to relative changes.

Reply:
As this is the bottom `axisline`, I think, then the width should be ~= the Axes width? Height is probably something like text (TTT) height + pad + linewidth, but that's a bit more difficult to put in.

Reply:
I'm ok pinning the current numbers as regression tests. But please add a comment on this so that we don't have to wonder later where these numbers came from.